CONTRIBUTING.md

```diff
@@ -18,13 +18,36 @@ Welcome! We're so excited that you're considering contributing to the official G
 ---
 
 ## Commit your update
+Please follow our project commit guide lines below:
+- The first line is the title of the message,
+it should be no longer than 63 characters.
+- The second line should be left empty.
+- The remaining consist of the body of the message,
+they should be no longer than 72 characters each.
+try to be descriptive as you can while keeping it simple.
+
+- Example:
+Adding an awesome feature.
+
+Adding feature which:
+1. makes cookies.
+2. makes you get A+ in all courses.
+3. create dank memes.
+in order to use each of the feature above just go to page ****,
+write the line "make_cookies()" where ever you want and walla, 
+cookies will appear.
+
 Commit the changes once you are happy with them. 
 
 ---
+## Review requirements 
+- Before revewing the task, please acknowledge its requirements and acceptance criteria.
+
 
+--- 
 ## Pull Request
 - When you're finished with the changes, create a pull request, also known as a PR.
-- Please note that at least two team members of the group should review and approve the pull request before asking the mentors to review and merge.
+- Please note that at least two team members of the group should quality review(NO LGTM) and approve the pull request before asking the mentors to review and merge.
 - The reviewer resolve the conversation to indicate that they are ok with the updated content.
 - If you run into any merge issues, check how to solve them and resolve merge conflicts and other issues.
```

Comment:
Stuff which I find missing:

```
- If your PR depends on any other PRs being merged, indicate so on the commit message or on the comments. Specifically, if it contains unmerged commits of any other PRs, indicate to the reviewers which commits are expected to be reviewed as part of your PR.
- Keep your PRs as small as possible.
- Ideally, a single PR should include a single commit.
- A PR's commit, needs to represent a transition in the code that completes a task (e.g. a new function, along with its unit tests) .
- Changes made as part of the review process (i.e. fixes to the content of the original commit) , should be added to the original commit (e.g. using `commit --amend`), rather than creating a new commit. 
```